main/modes/platformer/platformer_sounds.h

```diff
@@ -0,0 +1,2378 @@
+#ifndef _PLATFORMER_SOUNDS_H_
+#define _PLATFORMER_SOUNDS_H_
+
+//==============================================================================
+// Includes
+//==============================================================================
+#include <stdint.h>
+#include "musical_buzzer.h"
+
+//==============================================================================
+// Constants
+//==============================================================================
+
+//by FuryfulFawful
+static const song_t bgmDemagio =
```

Comment:
I really, really, really don't like variables in headers, even `const`. Put these in a C file, put the `extern` declarations in the header. And if you haven't merge the latest `main` into `platformer` and double check that the jukebox still works right.

Reply:
Will do